src/packaging/docker-services/generate-ssl-stores/docker-entrypoint.sh

```diff
@@ -35,7 +35,7 @@ set -ex
 
 CASSANDRA_KEYSTORE_PASSWORD=$1
 CASSANDRA_TRUSTSTORE_PASSWORD=$2
-
+WORKDIR=$PWD
```

Comment:
We can probably remove this line.

This is already defined as an environment variable in the [Dockerfile](https://github.com/thelastpickle/cassandra-reaper/blob/master/src/packaging/docker-services/generate-ssl-stores/Dockerfile#L18). It will already be set when the container is built. Hence, it will already be assigned a value when the _docker-entrypoint.sh_ executes. Only override this value if you really really needed to.